docs/area-owners.md

```diff
@@ -19,6 +19,7 @@ Note: Editing this file doesn't update the mapping used by `@dotnet-policy-servi
 | area-Codegen-AOT-mono                          | @steveisok           | @kotlarmilos                                                                             |                                                                                                                                                                                                                                                                                           |
 | area-CodeGen-coreclr                           | @JulieLeeMSFT        | @BruceForstall @dotnet/jit-contrib                                                       |                                                                                                                                                                                                                                                                                           |
 | area-Codegen-Interpreter-mono                  | @vitek-karas         | @BrzVlad @kotlarmilos                                                                    |                                                                                                                                                                                                                                                                                           |
+| area-Codegen-Interpreter-coreclr               | @vitek-karas         | @BrzVlad @janvorli                                                                       |                                                                                                                                                                                                                                                                                           |
```

Comment:
Nit: The list of areas looks sorted in alphabetical order. Move it before `area-Codegen-Interpreter-mono`?